tests/encryption.rs

```diff
@@ -4,25 +4,127 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
-use dusk_jubjub::{JubJubAffine, JubJubScalar, GENERATOR};
+use dusk_jubjub::{JubJubAffine, JubJubScalar, GENERATOR, GENERATOR_EXTENDED};
+use ff::Field;
 use rand_core::OsRng;
 
-use phoenix_core::{decrypt, encrypt, ENCRYPTION_EXTRA_SIZE};
+use phoenix_core::{aes, elgamal, PublicKey, SecretKey};
+
+#[cfg(feature = "zk")]
```

Comment:
you could add a 'tests/gadgets.rs' to avoid the `#[cfg(feature = "zk")` repetition.